src/Symfony/Component/Validator/Constraints/Negative.php

```diff
@@ -25,15 +25,21 @@ class Negative extends LessThan
 
     public function __construct($options = null)
     {
-        if (isset($options['propertyPath'])) {
-            throw new ConstraintDefinitionException(sprintf('The "propertyPath" option of the "%s" constraint cannot be set.', \get_class($this)));
+        if (null === $options) {
+            $options = array();
         }
 
-        if (isset($options['value'])) {
-            throw new ConstraintDefinitionException(sprintf('The "value" option of the "%s" constraint cannot be set.', \get_class($this)));
+        if (\is_array($options)) {
+            if (isset($options['propertyPath'])) {
+                throw new ConstraintDefinitionException(sprintf('The "propertyPath" option of the "%s" constraint cannot be set.', \get_class($this)));
+            }
+
+            if (isset($options['value'])) {
+                throw new ConstraintDefinitionException(sprintf('The "value" option of the "%s" constraint cannot be set.', \get_class($this)));
+            }
         }
 
-        parent::__construct(array_merge((array) $options, array('value' => 0)));
+        parent::__construct($options = array_merge((array) $options, array('value' => 0)));
```

Comment:
no need to re-assign $options here, also we lose the default option, due `array_merge((array) 'message value?', array(...))`

so this should be something like `if (!is_array($options)) { $options = array($this->getDefaultOption() => $options }`